### DISCUSSION TITLE
No answer on tickets

### DISCUSSION THREAD

**Discussion by Kebabipapi** [ovh/manager#4726] [Q&A] [answered]
We have been trying to get in touch with ovh for 2 days now and we have got no response! We have bought a dedicated machine and need help!

**Comment by antleblanc** [comment, marked as answer]
Hi @Kebabipapi!

Unfortunately there's nothing we can do regarding your order from here.
Please contact our support team who will be able to help you or refer to https://help.ovhcloud.com.

Thank you!
